### PR TITLE
Modifications to zero-shot classification branch

### PR DESCRIPTION
**Pull request by mathemakitten**
This just updates the PR to merge with main and bumps the task ID to 23.

### CODE REVIEW

File: .env.example
Comment (lewtun) [inline .env.example:1]: I'm not sure why these changes are appearing in your branch - maybe you need to rebase / force push?

**Review by lewtun — COMMENTED** [review]
Thanks for the hot fix @mathemakitten ! I think we just need to eliminate the changes coming from other branches / commits and then this is good to go

**Comment by mathemakitten** [comment]
I think it was just showing up with the diff from main because the feature
branch hadn't been merged main yet and I had to merge main w my branch
because I don't have repo access — I just saw you merged though :)

File: .env.example
Comment (mathemakitten) [reply .env.example:1]: oop sorry ok I just checked — it's because the feature branch I'm merging into isn't up-to-date. Can you merge main on the feat/add_zero_shot_classification branch? They should go away then! 

I've also added a small change which renames the task to text_zero_shot_classification as requested by the autotrain team.

Comment (lewtun) [reply .env.example:1]: Thanks, I've just brought `feat/add_zero_shot_classification` up to date with `main` - you'll now need to bring your current branch up to date with `feat/add_zero_shot_classification` to eliminate these finals diffs (I think!)

**Review by lewtun — APPROVED** [review]
LGTM once the unrelated diffs are removed - feel free to merge once they are!

**Comment by mathemakitten** [comment]
Thanks @lewtun — have fixed the merge conflict! Would you be able to merge this? I don't have write perms — `Only those with write access to this repository can merge pull requests.`

**Comment by lewtun** [comment]
> Thanks @lewtun — have fixed the merge conflict! Would you be able to merge this? I don't have write perms — `Only those with write access to this repository can merge pull requests.`

Done! I've also added you to the project so you should be able to merge future PRs :)

**Pull request merged**
